app/controllers/concerns/needs_hses_access_token.rb

```diff
@@ -0,0 +1,9 @@
+module NeedsHsesAccessToken
+  extend ActiveSupport::Concern
+
+  private
```

Comment:
You can kinda think of `include`ing a module in a class as copy/pasting all the methods into the class. The `private` there means that this method becomes a `private` method of the resulting class, rather than being a public method of the resulting class.

So, yes, the controller can invoke the method directly, but another class using the controller wouldn't be able to invoke it directly.